sdk/java/pulumi/src/main/java/com/pulumi/deployment/internal/DeploymentImpl.java

```diff
@@ -1132,8 +1132,8 @@ private CompletableFuture<Void> completeResourceAsync(
                         var data = response.data;
                         var dependencies = response.dependencies;
                         log.excessive(
-                                "Read response for resource: t=%s, name=%s, urn=%s, id=%s, remote=%s",
-                                resource.getResourceType(), resource.getResourceName(), urn, id, remote
+                                "Read response for resource: t=%s, name=%s, urn=%s, id=%s, remote=%s, data=%s",
+                                resource.getResourceType(), resource.getResourceName(), urn, id, remote, data
```

Comment:
Always wonder if printing `data` will leak secrets to logs or that's taken care of?

Reply:
very good question

Reply:
my assumption was that in excessive mode it is acceptable